ci/playbooks/deploy_edpm.yaml

```diff
@@ -10,6 +10,13 @@
         name: libguestfs-tools-c
         state: present
 
+    - name: Run make crc_storage
+      community.general.make:
+        target: crc_storage
+        chdir: "{{ install_yamls_basedir }}"
+        params:
+          OUT: "{{ install_yamls_basedir }}/out"
```

Comment:
@raukadah does it look good to you?

Reply:
@fao89 Thank you for adding me to the review. In our Zuul EDPM job, we do not run `make crc_storage` instead of that, we use[ `STORAGE_CLASS: crc-csi-hostpath-provisioner`](https://github.com/openstack-k8s-operators/install_yamls/blob/main/ci/playbooks/deploy_podified_openstack.yaml#LL41C11-L41C54). Is it possible to reuse the same for Ci purpose to store the dataplane logs also?

Reply:
I'm running this target because we need to create the PVC

Reply:
ok, let me merge it.